libs/fixtures/src/index.test.ts

```diff
@@ -34,6 +34,8 @@ test('has various election definitions', () => {
       "electionSampleDefinition",
       "electionSampleLongContent",
       "electionSampleLongContentDefinition",
+      "electionSampleNoSeal",
+      "electionSampleNoSealDefinition",
```

Comment:
Copied from `frontends/bmd` ([frontends/bmd/data/electionSampleNoSeal.json](https://github.com/votingworks/vxsuite/blob/main/frontends/bmd/data/electionSampleNoSeal.json) and [frontends/bmd/src/data/electionSampleNoSeal.json.ts](https://github.com/votingworks/vxsuite/blob/main/frontends/bmd/src/data/electionSampleNoSeal.json.ts)) for use in `@votingworks/ui` tests (next commit)